src/khoj/database/adapters/__init__.py

```diff
@@ -1031,6 +1037,52 @@ async def aget_advanced_conversation_config(user: KhojUser = None):
             return server_chat_settings.chat_advanced
         return await ConversationAdapters.aget_default_conversation_config(user)
 
+    @staticmethod
+    async def aget_server_webscraper():
+        server_chat_settings = await ServerChatSettings.objects.filter().prefetch_related("web_scraper").afirst()
+        if server_chat_settings is not None and server_chat_settings.web_scraper is not None:
+            return server_chat_settings.web_scraper
+        return None
+
+    @staticmethod
+    async def aget_enabled_webscrapers():
+        enabled_scrapers = []
+        server_webscraper = await ConversationAdapters.aget_server_webscraper()
+        if server_webscraper:
+            # Only use the webscraper set in the server chat settings
+            enabled_scrapers = [
+                (server_webscraper.type, server_webscraper.api_key, server_webscraper.api_url, server_webscraper.name)
```

Comment:
Should we create a simple pydantic object, like `WebScraperService` to make this code more maintainable? I find tuples mind-numbing 😅

Reply:
Good point. I've updated the code to use the WebScraper Django DB model directly. This has simplified this code